cmd/describe/machinepool/cmd.go

```diff
@@ -29,61 +30,68 @@ import (
 	"github.com/openshift/rosa/pkg/rosa"
 )
 
-var Cmd = &cobra.Command{
-	Use:     "machinepool",
-	Aliases: []string{"machine-pool"},
-	Short:   "Show details of a machine pool on a cluster",
-	Long:    "Show details of a machine pool on a cluster.",
-	Example: `  # Show details of a machine pool named "mymachinepool"" on a cluster named "mycluster"
-  rosa describe machinepool --cluster=mycluster --machinepool=mymachinepool`,
-	Run:  run,
-	Args: cobra.MaximumNArgs(1),
-}
+const (
+	use     = "machinepool"
+	alias   = "machine-pool"
+	short   = "Show details of a machine pool on a cluster"
+	long    = "Show details of a machine pool on a cluster."
+	example = `  # Show details of a machine pool named "mymachinepool"" on a cluster named "mycluster"
+  rosa describe machinepool --cluster=mycluster --machinepool=mymachinepool`
+)
 
 var args struct {
```

Comment:
@hunterkepley Take a look at the discussion in this thread: https://github.com/openshift/rosa/pull/1777#discussion_r1512694837

We're trying to find a pattern to avoid the global `args` struct to keep everything encapsulated. That discussion is our first attempt. Give it a try applying it here and let us know if it feels OK for you or if you can see a better way to approach it 👍 

cheers,

Rob